controllers/networking-calico/pkg/apis/calico/types_network.go

```diff
@@ -41,6 +50,9 @@ type NetworkConfig struct {
 	// https://docs.projectcalico.org/v2.2/reference/node/configuration#ip-autodetection-methods
 	// +optional
 	IPAutoDetectionMethod *string
+	// IPIP is the IPIP Mode for the IPv4 Pool (e.g. Always, Never, CrossSubnet)
+	// +optional
+	IPIP IPIP
```

Comment:
If this shall be optional then make it a pointer type:

```suggestion
	IPIP *IPIP
```

Reply:
OK